op-node/rollup/derive/engine_consolidate.go

```diff
@@ -47,8 +47,25 @@ func AttributesMatchBlock(rollupCfg *rollup.Config, attrs *eth.PayloadAttributes
 	if withdrawalErr := checkWithdrawalsMatch(attrs.Withdrawals, block.Withdrawals); withdrawalErr != nil {
 		return withdrawalErr
 	}
-	if envelope.ParentBeaconBlockRoot != attrs.ParentBeaconBlockRoot {
-		return fmt.Errorf("parent beacon block root does not match. expected %v. got: %v", attrs.ParentBeaconBlockRoot, envelope.ParentBeaconBlockRoot)
+	if err := checkParentBeaconBlockRootMatch(attrs.ParentBeaconBlockRoot, envelope.ParentBeaconBlockRoot); err != nil {
+		return err
+	}
+	return nil
+}
+
+func checkParentBeaconBlockRootMatch(attrRoot, blockRoot *common.Hash) error {
```

Comment:
Was this accidentally committed? I think that this is what is conflicting

Reply:
It was a stacked PR, but the dependency merged. Will rebase and drop the old dependency commit